content/400-reference/200-api-reference/300-environment-variables-reference.mdx

```diff
@@ -0,0 +1,144 @@
+---
+title: 'Environment variables reference'
+metaTitle: 'Prisma environment variables reference'
+metaDescription: 'This page gives an overview of all environment variables avalaible for use.'
+tocDepth: 2
+---
+
+<TopBlock>
+
+This document describes different environment variables and their use cases.
+
+</TopBlock>
+
+
+## Prisma Client
+
+### <inlinecode>DEBUG</inlinecode>
+
+`DEBUG` is used to enable debugging output in the Prisma Client.
+
+Example setting Prisma Client level debugging output:
+
+```terminal
+# enable only `prisma:client`-level debugging output
+export DEBUG="prisma:client"
+```
+
+See [Debugging](../../concepts/components/prisma-client/debugging) for more information.
+
+### <inlinecode>NO_COLOR</inlinecode>
+
+`NO_COLOR` if [truthy](https://developer.mozilla.org/en-US/docs/Glossary/Truthy) will activate the `colorless` setting for error formatting and strip colors from error messages.
+
+See [Formatting via environment variables](../../concepts/components/prisma-client/working-with-prismaclient/error-formatting#formatting-via-environment-variables) for more information.
+
+## Prisma Studio
+
+### <inlinecode>BROWSER</inlinecode>
+
+`BROWSER` is for Prisma Studio to force which browser it should be open in, if not set it will open in the default browser. It's also used as a flag when starting the studio from the CLI.
+
+```js
+BROWSER=firefox prisma studio --port 5555
+```
+
+```terminal
+prisma studio --browser firefox
+```
+
+See [Studio](./command-reference/#studio) documentation for more information.
+
+## Prisma CLI
+
+### <inlinecode>PRISMA_HIDE_UPDATE_MESSAGE</inlinecode>
+
+`PRISMA_HIDE_UPDATE_MESSAGE` is used to hide the update notification message that is shown when a newer Prisma CLI version is available. It's a truthy value.
+
+```js
+const shouldHide = process.env.PRISMA_HIDE_UPDATE_MESSAGE
+```
+
+### <inlinecode>PRISMA_GENERATE_SKIP_AUTOINSTALL</inlinecode>
+
+`PRISMA_GENERATE_SKIP_AUTOINSTALL` can be set to a truthy value to skip the auto-install of `prisma` CLI and `@prisma/client` dependencies (if they are missing) when using the `prisma generate` command.
+
+### <inlinecode>SKIP_GENERATE</inlinecode>
+
+`SKIP_GENERATE` can be set to a truthy value to skip running `prisma generate` on npm's postinstall lifecycle hook.
+
+## Proxy environment variables
+
+The Prisma CLI supports custom HTTP proxies. These can be helpful to use when working behind a corperate firewall. See [Using a HTTP proxy for the CLI](../../concepts/components/prisma-cli/installation/#using-a-http-proxy-for-the-cli) for more information.
+
+### <inlinecode>NO_PROXY</inlinecode>
+
+`NO_PROXY` is a comma-seperated list of hostnames, IP addresses or IP ranges that do not require a proxy.
```

Comment:
Where did you see that it could be `IP ranges` @molebox not sure this is implemented 🤔

Reply:
I read it when googling about it. It could be that was just implemented in their solution (those that wrote the article) Lemme see if its in my history

Reply:
First [google result](https://docs.openshift.com/enterprise/3.2/install_config/http_proxies.html) I made the assumption that it was standard. I guess not if you are saying otherwise

Reply:
Should i remove this then?

Reply:
```suggestion
`NO_PROXY` is a comma-separated list of hostnames or IP addresses that do not require a proxy.
```
Yes from what I see IP ranges are not supported in our code